scripts/check-env-example.mjs

```
@@ -0,0 +1,37 @@
+  ),
+);
+
+let failed = false;
+for (const file of COMPOSE) {
+  const used = new Set(
+    [...readFileSync(file, 'utf8').matchAll(/\$\{([A-Z][A-Z0-9_]*)/g)].map((m) => m[1]),
+  );
+  const missing = [...used].filter((v) => !declared.has(v)).sort();
+  if (missing.length) {
+    failed = true;
+    console.error(`${file}: not documented in .env.example — ${missing.join(', ')}`);
```

ESLint

[error] 'console' is not defined.

(no-undef)